### PR TITLE
documentation: tutorials: add basic setup tutorial

### PR DESCRIPTION
**Pull request by joaoseckler**
This is part of #244.

Signed-off-by: João Seckler <jseckler@riseup.net>

### CODE REVIEW

**Review by magalilemes — CHANGES_REQUESTED** [review]
Very nice writing and explanation :-)

File: documentation/tutorials/setup.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+First of all, let's download kw::
+
+  git clone https://github.com/kworkflow/kworkflow.git
+
+We recommend checking out the unstable branch, to get the latest version
+of software::
+
+  git checkout unstable
+
+Alternatively, you can check if kw is available via package manager. For
```
Comment (alan-barzilay) [inline documentation/tutorials/setup.rst:23]: maybe "is available in your distribution package manager"

File: documentation/tutorials/setup.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  cd kworkflow
+
+Then install kw::
+
+  ./setup.sh --install
+
+Actually, in this folder you can see three executables: ``run_tests.sh``,
+``setup.sh`` and ``kw``. The file ``run_tests.sh`` is used by kw's developers to
+run it's test suite. ``setup.sh`` is used to install kw to your system, like we
+just did. ``kw`` is kw's main executable, but it should not be called from this
```
Comment (alan-barzilay) [inline documentation/tutorials/setup.rst:43]: maybe a quick explanation of why it shouldn´t be run like this even though it is possible? maybe mention the lack of tab completion and how this use is more directed for developers

File: documentation/tutorials/setup.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+  kw help
+
+You can check out more details in :ref:`installing kw<install-and-remove-kw>`.
+After installing, you should check that everything is working as expected. Try
+running::
+
+  kw version
+
+Maybe the autocomplete feature of your shell will not have been loaded.
```
Comment (alan-barzilay) [inline documentation/tutorials/setup.rst:56]: how about: "kw's tab-completion may not have been loaded by your shell after installation."

File: documentation/tutorials/setup.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+To load it, simply restart the shell.
+
+Configuring kw
+--------------
+Kw works with three levels of configuration files: global, user, and local. The
+`global` configuration file contains the default values for options and may be
+in ``~/.local/etc/kw/kworkflow.config`` or in
+``/etc/kworkflow/kworkflow.config``, depending on the installation method. The
```
Comment (alan-barzilay) [inline documentation/tutorials/setup.rst:64]: off topic but, I had no idea about this haha which installation method leads to which location?

File: documentation/tutorials/setup.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+Next steps
+----------
+Next, you might want to :ref:`build your kernel <buildlinux>` using kw.
+Then, you might want to deploy (see ``kw deploy``) it, that is, install
+it in some machine. Unless you know what you are doing, you shouldn't do
+that to you own main machine (locally). Instead, configure a virtual
+machine following `this tutorial
+<https://flusp.ime.usp.br/others/use-qemu-to-play-with-linux/>`_.
+Another alternative is to use an external machine, with which you can
+communicate with ssh. By the way, these three possible ways to deploy a
```
Comment (alan-barzilay) [inline documentation/tutorials/setup.rst:128]: how about "through" instead of "with"

File: documentation/tutorials/setup.rst (hunk 4 above)
Comment (joaoseckler) [reply documentation/tutorials/setup.rst:64]: In the near future, `apt install kworkflow` will lead to this!

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]

**Comment by rodrigosiqueira** [comment]
Hi @joaoseckler ,
Could you update this PR based on all comments?

**Comment by rodrigosiqueira** [comment]
Merged! Thanks

**Pull request closed without merging**